config/api.yml

```diff
@@ -484,6 +484,33 @@
       :get:
       - :name: read
         :identifier: miq_cloud_networks_view
+  :cloud_tenants:
+    :description: Cloud Tenants
+    :identifier: cloud_tenant
+    :options:
+    - :collection
+    - :subcollection
+    :verbs: *gp
+    :klass: CloudTenant
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: cloud_tenant_show_list
+      :post:
+      - :name: query
+        :identifier: cloud_tenant_show_list
+    :resource_actions:
+      :get:
+      - :name: read
+        :identifier: cloud_tenant_show
+    :subcollection_actions:
+      :get:
+      - :name: read
+        :identifier: cloud_tenant_show
```

Comment:
Turns out that this seems correct, at least for purposes of passing the providers_spec.rb.  I'm following the existing pattern used by load balancers.